src/scheduled_tasks.rs

```diff
@@ -155,6 +166,48 @@ fn clear_old_activities(conn: &mut PgConnection) {
   }
 }
 
+/// overwrite posts and comments 30d after deletion
+fn overwrite_deleted_posts_and_comments(conn: &mut PgConnection) {
+  // TODO: localize this at display time via lemmy-translations
+  let perma_deleted = "*Permanently Deleted*";
```

Comment:
You can move this to a constant so it can be reused across the different functions. No need for the todo.